modules/planning/common/frame.h

```diff
@@ -169,6 +170,13 @@ class Frame {
 
   perception::TrafficLight GetSignal(const std::string &traffic_light_id) const;
 
+  // tuple<exist_pull_over_position, x, y, theta>
+  std::tuple<bool, double, double, double> pull_over_info() {
```

Comment:
Return a const reference. You save a copy if user only wants an immutable reference.

Reply:
fixed this.